### PR TITLE
Add timestamps to edges

### PR DESCRIPTION
**Pull request by berggren**
Fixes: #470

### CODE REVIEW

File: timesketch/lib/experimental/win_services.py [hunk 1]
```diff
@@ -69,20 +69,34 @@ def win_services(sketch_id):
             sketch_id=sketch_id, query=u'event_identifier:7045'):
         data = event[u'_source'][u'xml_string']
         res = parse_xml(data)
-        res.extend((event.get(u'_index'), event.get(u'_id')))
+        res.extend(
+            (
+                event[u'_source'].get(u'timestamp'),
+                event.get(u'_index'),
+                event.get(u'_id')
```
Comment (franekp) [inline timesketch/lib/experimental/win_services.py]: trailing comma

File: timesketch/lib/experimental/win_logins.py [hunk 2]
```diff
@@ -153,12 +158,13 @@ def win_logins(sketch_id):
...
+        src_ws, user, dst_ws, method, timestamp, es_index_name, es_id = event
         result.append({
             u'user': user,
             u'src': src_ws,
             u'dst': dst_ws,
             u'method': method,
+            u'timestamp': timestamp,
             u'es_index_name': es_index_name,
             u'es_query': u'_index:{} AND _id:{}'.format(es_index_name, es_id)
         })
```
Comment (franekp) [inline timesketch/lib/experimental/win_logins.py]: trailing comma

File: timesketch/lib/experimental/win_logins.py [hunk 3]
```diff
@@ -89,20 +89,24 @@ def parse_xml(self, xml):
...
 
-        event_tuple = (
+        event_list = [
             event_container[u'src_ip'],
             event_container[u'src_hostname'],
             event_container[u'dst_hostname'],
             event_container[u'username'],
-            event_container[u'logon_type'],)
-        return event_tuple
+            event_container[u'logon_type']]
```
Comment (franekp) [inline timesketch/lib/experimental/win_logins.py]: trailing comma

Comment (berggren) [reply timesketch/lib/experimental/win_logins.py]: ack

File: timesketch/lib/experimental/win_logins.py (hunk 2 above)
Comment (berggren) [reply timesketch/lib/experimental/win_logins.py]: ack

File: timesketch/lib/experimental/win_services.py (hunk 1 above)
Comment (berggren) [reply timesketch/lib/experimental/win_services.py]: ack

**Pull request merged**